src/psyclone/tests/parse/module_info_test.py

```diff
@@ -401,28 +405,24 @@ def test_module_info_coverage_a(tmpdir, monkeypatch):
         fout.write(SOURCE_DUMMY)
 
     # We create a dummy
-    node = Node("Dummy")
     module_info: ModuleInfo = ModuleInfo(
             "my_mod",
-            FileInfo(filename),
-            node
+            FileInfo(filename)
         )
 
-    # view() creates also a list of all modules based on fparser!!!
-    module_info.view()
-    assert module_info is not None
+    module_info.view_tree()
```

Comment:
Please check that the output of `view_tree` is what you expect.

Reply:
Makes sense, I added it.